index.html

```diff
@@ -219,29 +219,29 @@
 var sum = SumOf(42, 597, 12);
 
 // SumOf() can also be written in a shorter way:
-int SumOf(params int[] numbers) => numbers.Sum();</code></pre></div></div></div><div class="case"><div class="name">Lambdas</div><div class="pair"><div class="card"><div class="lang">Kotlin</div><pre class="code"><code>fun containsEven(numbers: List&lt;Int>) = numbers.any { it % 2 == 0 }</code></pre></div><div class="card"><div class="lang">C#</div><pre class="code"><code>bool ContainsEven(List&lt;int> numbers) => numbers.Any(e => e % 2 == 0);</code></pre></div></div></div><div class="case"><div class="name">Higher-Order Functions - Return a Function</div><div class="pair"><div class="card"><div class="lang">Kotlin</div><pre class="code"><code>fun makeIncrementer(): (Int) -> Int {
+int SumOf(params int[] numbers) => numbers.Sum();</code></pre></div></div></div><div class="case"><div class="name">Lambdas</div><div class="pair"><div class="card"><div class="lang">Kotlin</div><pre class="code"><code>fun containsEven(numbers: List&lt;Int>) = numbers.any { it % 2 == 0 }</code></pre></div><div class="card"><div class="lang">C#</div><pre class="code"><code>bool ContainsEven(List&lt;int> numbers) => numbers.Any(e => e % 2 == 0);</code></pre></div></div></div><div class="case"><div class="name">Higher-Order Functions - Return a Function</div><div class="pair"><div class="card"><div class="lang">Kotlin</div><pre class="code"><code>fun makeIncrementor(): (Int) -> Int {
```

Comment:
Thanks for the typo fixes!

index.html file is generated from the template and code files, e.g. this part with `makeIncrementor()` fix can be found from a file: https://github.com/ttu/kotlin-is-like-csharp/blob/1326faf5a49283dcfb9ea7c99a3120d236945e52/code/hof-return.kt#L1

Would you like to make the fixes to the code files? If you don't have time to make the fixes, you can also leave this PR open and I will make the changes at some point.

Reply:
I just looked into the actual difference between the two and it appears that `makeIncrementer()` is actually correct in this case. At least for Kotlin. I believe both `MakeIncrementer()` and `MakeIncrementor()` don't follow C#'s guidelines. Since Kotlin prefers `makeIncrementer()`, I'll revert my change for this in `index.html`. Nice catch!

Reply:
Alright, I've fixed the issue 👍

Reply:
Thanks I will close this PR and move changes to code files.